### DISCUSSION TITLE
Can I use Mantine Dates v7 with Mantine v6?

### DISCUSSION THREAD

**Discussion by sowizz** [mantinedev/mantine#7119] [Q&A]
As in the title. I need timezone support, which I believe was added in v7, and cannot migrate the whole app to v7. Is it possible?

**Comment by rtivital** [comment]
No, it is not possible

**Reply by sowizz** [reply]
@rtivital, then a followup question:

I'm using the DatePickerInput component. I'm setting a `value`. Why does the input display the correct date, but the Calendar component highlights something different? I've set the `date` property as well, but it seems to be ignored. Is this a known bug?
![image](https://github.com/user-attachments/assets/73ad60c9-0dee-4e74-a364-a982fe696489)

**Reply by rtivital** [reply]
Probably related to time specified in the Date object, make sure to set hours, minutes and seconds to 0
